mscore/tourhandler.cpp

```diff
@@ -0,0 +1,460 @@
+#include "tourhandler.h"
```

Comment:
I really like how few changes are required elsewhere in the code, so most of the work is done here in this tourhandler.cpp